Assets/AnkrSDK/Scripts/Core/Data/EventFilterRequest.cs

```diff
@@ -1,11 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Nethereum.ABI.FunctionEncoding.Attributes;
 using Nethereum.RPC.Eth.DTOs;
 
 namespace AnkrSDK.Core.Data
 {
-	public class EventFilterRequest<TEvDTO>
+	public class EventFilterRequest<TEvDto>
 	{
-		public TEvDTO request { get; set; }
-		public BlockParameter fromBlock { get; set; }
-		public BlockParameter toBlock { get; set; }
+		private class Topic
+		{
+			public string Name { get; set; }
+			public string Alias { get; set; }
+			public object Value { get; set; }
+		}
+
+		public BlockParameter FromBlock { get; set; }
+		public BlockParameter ToBlock { get; set; }
```

Comment:
move public below private readonly

Reply:
Can you clarify why? Last time you said about that sequence if i'm right:
public
private
private readonly

Reply:
it is in reverse order
1 private readonly
2 private
3 public